src/sxastarter/src/components/play-travel/PageCard.tsx

```diff
@@ -66,16 +67,21 @@ export const Default = ({ params, fields }: PageCardProps): JSX.Element => {
             </Box>
           </Box>
           <Flex className="pt-0 md:pt-5 pb-5 px-5">
-            <Button
-              bgGradient="linear(135deg, #8482FF, #7723FE)"
-              className="text-sm mb-3 basis-full"
-              _hover={{ bg: '#7723FE' }}
-              colorScheme="purple"
+            <Link
+              className="text-sm mb-3 basis-full flex justify-center link-button items-center transition-shadow"
+              href={sitecoreContext.pageEditing ? undefined : fields.data.item.link.url}
```

Comment:
It will be better if you change `undefined` to empty string. Because href will be like `href="undefined"` it's incorrect url

Reply:
no, if it's an empty string, it will generate the following html which includes an empty href
<a class="chakra-link text-sm mb-3 basis-full flex justify-center link-button items-center transition-shadow css-1yutgmj" **href=""**>Learn more</a>

in my case it won't even add href attribute to <a>

![image](https://user-images.githubusercontent.com/16080426/218193837-3206947e-93c4-42a5-93f2-02117d975d4b.png)


stackoverflow sasy the same
![image](https://user-images.githubusercontent.com/16080426/218193955-ff1357ac-c123-493a-a0ca-d951f091e7f2.png)

https://stackoverflow.com/questions/32232659/react-inline-conditionally-pass-prop-to-component

Reply:
Okay, I checked it - works